GSA_Adapter/CRUD/Read.cs

```diff
@@ -408,6 +410,37 @@ public List<SpacerProperty> ReadSpacerProperties(List<string> ids = null)
                 return proArr.Where(x => ids.Contains(x.Split(',')[1])).Select(x => Convert.FromGsaSpacerProperty(x)).ToList();
         }
 
+        /***************************************/
+
+        public List<PointLoad> ReadNodeLoads(List<string> ids = null)
+        {
+            List<PointLoad> pointLoads = new List<PointLoad>();
+
+            double[] unitFactors = GetUnitFactors();
+
+            int nodeLoadsCount = m_gsaCom.GwaCommand("HIGHEST, LOAD_NODE.2");
+            List<string> nodeLoadList = new List<string>();
+            for (int i = 0; i < nodeLoadsCount; i++)
+            {
+                string nodeLoad = m_gsaCom.GwaCommand("GET, LOAD_NODE.2," + (i + 1)).ToString();
+                if (!string.IsNullOrWhiteSpace(nodeLoad))
+                    nodeLoadList.Add(nodeLoad);
+            }
+
+            List<Loadcase> lCaseList = ReadLoadCases();
+            Dictionary<string, Loadcase> lCases = lCaseList.ToDictionary(x => x.Number.ToString());
+
+            List<Node> nodeList = ReadNodes();
+            Dictionary<string, Node> nodes = nodeList.ToDictionary(x => GetAdapterId<int>(x).ToString());
```

Comment:
Could make use of the recently added caching system here:

```suggestion
            Dictionary<int, Loadcase> lCases = GetCachedOrReadAsDictionary<int, Loadcase>();
            Dictionary<int, Node> nodes = GetCachedOrReadAsDictionary<int, Node>();
```

Reply:
Thanks! Will do a commit covering this and related changes